backend/src/utils/notebookUtils.ts

```diff
@@ -68,6 +69,35 @@ export const getRoute = async (
   return kubeResponse.body as Route;
 };
 
+export const getServiceMeshGwHost = async (
+  fastify: KubeFastifyInstance,
+  namespace: string,
+): Promise<string> => {
+  const kubeResponse = await fastify.kube.coreV1Api.readNamespace(namespace).catch((res) => {
+    const e = res.response.body;
+    const error = createCustomError('Error getting Namespace', e.message, e.code);
+    fastify.log.error(error);
+    throw error;
+  });
+
+  const body = kubeResponse.body as unknown;
+  const typedResponse = body as V1Namespace;
+
+  const annotations = typedResponse.metadata?.annotations;
```

Comment:
Why? `readNamespace` is already returning a typed response, you don't need to cast anything here, cast as little as possible, in this case you can have:


` const annotations = kubeResponse.body.metadata?.annotations;`

Reply:
Good point, addressed here 

https://github.com/opendatahub-io/odh-dashboard/pull/1088/commits/ca51ef37fc25d64af1321ebfa40478a10b4fafc7